### PR TITLE
better logging

### PR DESCRIPTION
**Pull request by Nyameliaaaa**
this makes the logging in sync with interactions

might add  other things.

### CODE REVIEW

**Review by hkubus — DISMISSED** [review]

**Comment by Nyameliaaaa** [comment]
perhaps i should also make `Logger` static?

**Comment by SuperchupuDev** [comment]
> perhaps i should also make `Logger` static?

if it works +1 on that

**Comment by Nyameliaaaa** [comment]
> > perhaps i should also make `Logger` static?
> 
> if it works +1 on that

it  could work but you wouldnt be able to log with `FluorineClient#logger`

**Comment by Nyameliaaaa** [comment]
> > perhaps i should also make `Logger` static?
> 
> if it works +1 on that

alright impemented in 8adc6fe

File: src/classes/Client.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Client, Collection, ColorResolvable, Intents } from 'discord.js';
 import r from 'rethinkdb';
-import Logger from './Logger';
+import exposedLoggerProps from './Logger';
```
Comment (SuperchupuDev) [inline src/classes/Client.ts]:
```suggestion
import { Logger } from './Logger';
```

File: src/classes/Client.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default class FluorineClient extends Client {
     footer: string;
     color: ColorResolvable;
     devs: string[];
-    logger: Logger;
+    logger: typeof exposedLoggerProps;
```
Comment (SuperchupuDev) [inline src/classes/Client.ts]:
```suggestion
    logger: typeof Logger;
```

File: src/classes/Client.ts [hunk 3]
```diff
@@ -61,26 +61,25 @@ export default class FluorineClient extends Client {
             '478823932913516544',
             '348591272476540928'
         ];
-        this.logger = new Logger();
+        this.logger = exposedLoggerProps;
```
Comment (SuperchupuDev) [inline src/classes/Client.ts]:
```suggestion
        this.logger = Logger;
```

File: src/classes/Logger.ts [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
     }
 }
+
+const exposedLoggerProps = {
+    log: Logger.log,
+    error: Logger.error,
+    warn: Logger.warn
+};
+
+export default exposedLoggerProps;
```
Comment (SuperchupuDev) [inline src/classes/Logger.ts]:
```suggestion
```

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]

**Review by SuperchupuDev — APPROVED** [review]

**Review by hkubus — APPROVED** [review]

**Pull request merged**